compiler/src/test/kotlin/io/spine/protodata/plugin/JustTest.kt

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2021, TeamDev. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package io.spine.protodata.plugin
+
+import com.google.common.truth.Truth.assertThat
+import io.spine.base.Identifier.newUuid
+import io.spine.protodata.test.ProjectCreated
+import org.junit.jupiter.api.Test
+
+class `Just should` {
+
+    @Test
+    fun `store one value`() {
+        val just = Just(validEvent)
+        assertThat(just)
+            .containsExactly(validEvent)
+    }
+
+    @Test
+    fun `initialize via static method`() {
+        val just = Just.just(validEvent)
+        assertThat(just)
+            .containsExactly(validEvent)
+    }
+}
+
+private val validEvent = ProjectCreated
+    .newBuilder()
+    .setId(newUuid())
+    .build()
```

Comment:
Why not `vBuild` or `buildPartial`?
```suggestion
private val validEvent = ProjectCreated.newBuilder()
    .setId(newUuid())
    .build()
```

Reply:
I've changed the method to `vBuild()` for now. However, please keep in mind that in future, `vBuild()`-s will die off and `build()` will validate messages.